packages/samples/angular/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "dev": "npm run copy:assets && ng serve",
+    "dev": "npm run build:assets && dev:angular",
+    "dev:angular": "ng serve",
     "link:angular": "node scripts/link-hoisted-dependencies.mjs",
-    "watch": "ng build --watch --configuration development",
```

Comment:
`watch` seems redundant since its purpose is pretty much covered by `dev`, so I removed it.